tests/unit/contract_id_test.py

```
@@ -282,45 +289,49 @@ def test_to_evm_address():
+
 def test_validate_checksum_success(client):
     """Should pass checksum validation when checksum is correct."""
     contract_id = ContractId.from_string("0.0.1-dfkxr")
     contract_id.validate_checksum(client)
 
+
 def test_validate_checksum_failure(client):
     """Should raise ValueError if checksum validation fails."""
     contract_id = ContractId.from_string("0.0.1-wronx")
 
     with pytest.raises(ValueError, match="Checksum mismatch for 0.0.1"):
```

Ruff (0.14.13)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)